### ISSUE TITLE
Alias order doesn't respect case.

### ISSUE THREAD

**Issue by alecho** [rrrene/credo#1007]
### Precheck

* Proposals for new features should be submitted via: https://github.com/rrrene/credo-proposals
* For bugs, please do a quick search and make sure the bug has not yet been reported here

### Environment

* Credo version 1.6.7:
* Erlang/Elixir version Elixir 1.13.4 (compiled with Erlang/OTP 23):
* Operating system: macOS 12.6

### What were you trying to do?

Lint alias order.

### Expected outcome

This shouldn't warn as this is how vim's `sort` and Recode's autocorrect wants to sort these.

```elixir
    defmodule Test do
      alias MyApp.AlphaBravoCharlie
      alias MyApp.AlphaBravoalpha
    end

```

### Actual outcome

Credo complains with

`The alias `MyApp.AlphaBravoCharlie` is not alphabetically ordered among its group.`

**Comment by alecho** [comment]
I opened a PR with just a test: #1006 Sorry I couldn't come up with a fix in the time allowed.

**Comment by rrrene** [comment]
Just for completeness sake (currently falling behind on commenting, sorry): 

I commented in the referenced PR the current implementation works as expected, but we could provide an alternative ordering option via a check parameter.

**Comment by alecho** [comment]
@rrrene I suspected that might be the suggested solution. If I get some time I'll take a stab at it.

**Comment by rrrene** [comment]
Thx for reporting this! :+1: It should be be fixed in the latest version of Credo.

If it is not, please feel free to re-open this issue!
